src/app.ts

```diff
@@ -13,14 +14,66 @@ interface TemperatureSummary {
   average: number
 }
 
+interface ProcessedReading {
+  city: string
+  time: Date
+  readings: number[]
+}
+
+const processedReadings: ProcessedReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readings.forEach((reading: TemperatureReading) => {
```

Comment:
`reading: TemperatureReading` can be inferred since `readings` is a `TemperatureReading[]`